components/Componenten/form/form.njk

```diff
@@ -1,11 +1,17 @@
-<form {{ className(formModifier) }}>
+{% if skipFormTag !== "false" %}
+  <form {{ className(formModifier) }}>
+{% endif %}
   {% for fieldset in fieldsets %}
-    <fieldset>
+    {% if fieldset.legend.length > 0 %}
```

Comment:
Zie ook mijn opmerking bij de yml. Misschien moeten we het model gewoon wat flexibeler maken in plaats van al dit soort checkjes in te moeten bouwen?